var/spack/repos/builtin/packages/py-ford/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+
```

Comment:
```suggestion
```

Reply:
Suggest removing this comment and have one blank line between the License Identifier comment and first import.